src/App.tsx

```diff
@@ -21,17 +21,25 @@ import Rooms from './Components/Rooms/Rooms'
 import AddRoom from './Components/AddRoom/AddRoom'
 import Navbar from './Components/Navbar/Navbar'
 import { AuthContext } from './Context/AuthContext'
-import { useContext } from 'react'
+import { useContext, useState } from 'react'
+import RoomDetails from './Components/RoomDetails/RoomDetails'
 import ChangePassword from './Components/ChangePassword/ChangePassword'
 import Facilities from './Components/Facilities/Facilities'
 import { ToastContainer } from 'react-toastify'
 import ProtectedRoute from './Components/shared/ProtectedRoute/ProtectedRoute'
 import AddAds from './Components/AddAds/AddAds'
+import AvilableRooms from './Components/AvilableRooms/AvilableRooms'
+import MyDate from './Components/Date/Date'
 
 
 function App() {
  
   let {userData,saveUserData,role}:any=useContext(AuthContext)
+  const [count, setCount] = useState(0);
```

Comment:
I don't understand the reason for adding the state variable, as it is not used in the application.